source/blog/2018-12-14-the-ember-times-issue-77.md

```diff
@@ -25,8 +25,10 @@ Leave your thoughts on how boolean arguments could be implemented for Ember comp
 
 ---
 
-## [SECTION TITLE](#section-url)
+## [Ember 3.6 released 🚀](https://emberjs.com/blog/2018/12/13/ember-3-6-released.html)
+Ember 3.6 is out and ready to use. This release contains the last step of the **Native Class Constructor Update RFC**.  This introduces native classes in Ember but with some caveats so remember to read the [release post]([Ember.js - Ember 3.6 Released](https://emberjs.com/blog/2018/12/13/ember-3-6-released.html#toc_new-features-2)) carefully before refactoring your code base.
```

Comment:
I think the link for the release post is somewhat doubled regarding formatting. Would it be

```
[release post](https://emberjs.com/blog/2018/12/13/ember-3-6-released.html#toc_new-features-2)
```